src/components/totals/FundsRaised/__tests__/FundsRaised-test.js

```diff
@@ -239,4 +239,20 @@ describe('FundsRaised', function() {
     });
   });
 
+  describe('takes groupValues property', function() {
```

Comment:
I've been using this form:

```
describe('props.groupValues', function () {})
```

What do you reckon?

`describe`s seem to be for "this is what I'm about to test", so noun-y things like the module, function or property. And `context`s and `when`s for "this is a new set of conditions for the thing I am currently testing".

```
describe('props.foo', function () {
  when('false', function () {
    it('behaves some way', function () {})
  })
})
```

@alexander-bobin @benoneal @kizza. Curious to know what you guys think.